test/e2e/tests/tokens/token-details.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { strict as assert } from 'assert';
+import { Mockttp } from 'mockttp';
+import { CHAIN_IDS } from '../../../../shared/constants/network';
+import { toChecksumHexAddress } from '../../../../shared/modules/hexstring-utils';
+import { formatCurrency } from '../../../../ui/helpers/utils/confirm-tx.util';
+import FixtureBuilder from '../../fixture-builder';
+import {
+  clickNestedButton,
+  defaultGanacheOptions,
+  unlockWallet,
+  withFixtures,
+} from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+
+describe('Token Details', function () {
+  it('shows details for an ERC20 token', async function () {
```

Comment:
good job ! can we add an e2e tests when data are not available ?

Reply:
Added more unit and e2e tests